doc/sphinx/ui/mainviews/charview.rst

```diff
@@ -499,10 +499,10 @@ including the start and end points of the line itself. If there are more than
 two intersections then the x, y, and total distance between the first and last
 intersections are shown before the point list.
 
-[0] indicates the starting point of the measure line, and the (x-y) co-ordinates
+[0] indicates the starting point of the measure line, and the (x-y) coordinates
 of that point. Initially [1] is the end point; when it intersects with one
 spline then [1] becomes the first intersection point and [2] becomes the end
-point, and so on. The co-ordinates are followed by the x and y distances and
```

Comment:
This is a formatting choice, not a typographical error.

Reply:
Reverted in 1f5831e